### PR TITLE
Add Gpart & Sharedmimeinfo Packages

### PR DESCRIPTION
**Pull request by beliefless-dogmasless**
Hy,

Both work 'fine' on arm.

### CODE REVIEW

File: packages/gpart.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'package'
+
+class Gpart < Package
+  description 'Partition table rescue/guessing tool. https://github.com/baruch/gpart'
+  homepage 'https://github.com/baruch/gpart/releases'
```
Comment (uberhacker) [inline packages/gpart.rb]:
```
- homepage 'https://github.com/baruch/gpart/releases'
+ homepage 'https://github.com/baruch/gpart'
```

File: packages/sharedmimeinfo.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Sharedmimeinfo < Package
+  description 'The Shared MIME-info Database specification. https://freedesktop.org/wiki/Software/shared-mime-info'
```
Comment (uberhacker) [inline packages/sharedmimeinfo.rb]:
```
- description 'The Shared MIME-info Database specification. https://freedesktop.org/wiki/Software/shared-mime-info'
+ description 'The Shared MIME-info Database specification.'
```

File: packages/gpart.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+require 'package'
+
+class Gpart < Package
+  description 'Partition table rescue/guessing tool. https://github.com/baruch/gpart'
```
Comment (uberhacker) [inline packages/gpart.rb]:
```
- description 'Partition table rescue/guessing tool. https://github.com/baruch/gpart'
+ description 'Partition table rescue/guessing tool.'
```

**Comment by uberhacker** [comment]
Thank you for the new packages @beliefless-dogmasless!  Please see inline comments.

**Comment by beliefless-dogmasless** [comment]
The 'only problem' with that method, is rarely package have both information(on the same page:downloads and information about the package). So, if I keep, one in the description(information about the package), and one in the homepage(downloads, it leaves 'people' with two information, and the advantage to check relatively rapidly the version, and 'make them' compile/and propose new version.

**Comment by uberhacker** [comment]
Rather than combining the homepage url in with the description, why not separate it out in another field?  Should we propose a new metadata value?  You can also get the url that you have stored in homepage from the source_url by removing the tarball from the end.

**Comment by beliefless-dogmasless** [comment]
<a href="http://fr.tinypic.com?ref=21llgnb" target="_blank">Image and video hosting by TinyPic</a>
It's 'neat' to have the address directly. Then, if people want to collaborate, they don't have to search 'a lot of time' to find it.

**Comment by uberhacker** [comment]
What is wrong with `crew search gpart -v`?

**Comment by beliefless-dogmasless** [comment]
It's perfect, no 'they' 'have all' the information they need to participate.
<a href="http://fr.tinypic.com?ref=258v1bl" target="_blank">Image and video hosting by TinyPic</a>

**Pull request merged**